mmv1/templates/terraform/examples/bigquery_job_copy.tf.tmpl

```diff
@@ -92,8 +93,8 @@ data "google_project" "project" {
   project_id = "{{index $.TestEnvVars "project"}}"
 }
 
-resource "google_project_iam_member" "encrypt_role" {
-  project = data.google_project.project.project_id
+resource "google_kms_crypto_key_iam_member" "encrypt_role" {
```

Comment:
Can we put this iam member resource in `.ci/infra/terraform/main.tf` instead of here? If it's part of the test run, it will get cleaned up when it finishes and potentailly cause simultaneous runs to fail.

Reply:
That's fine, for now, imo- we want to record the permission grants in our examples. This is the exclusive user so far.

Reply:
(if we have a method to both apply permissions in configs *and* add the permission there, glad to follow that instead though- just don't want to take on the larger correction in place of the immediate problem of fixing the test)

Reply:
What ensures that the key used in this test is not also used in other tests?

Reply:
Nothing structurally. I'm not _super_ worried about it happening because it's a relatively uncommon feature, and this is currently the sole user. If this were not an example test I think it would worth correcting now, but I think we need to solve for a way to show permission grants in examples while actually handling them OOB.